pkg/framework/didresolver/api.go

```diff
@@ -6,6 +6,7 @@ SPDX-License-Identifier: Apache-2.0
 package didresolver
 
 import (
+	"errors"
```

Comment:
Should import xerrors and alias to errors.

Reply:
Fixed